nurseconnect/forms.py

```diff
@@ -0,0 +1,60 @@
+from django import forms
+from django.forms import Form
+from django.utils.translation import ugettext_lazy as _
+from nurseconnect.constants import GENDERS
+from molo.profiles.forms import RegistrationForm, EditProfileForm
+
+
+class NurseConnectRegistrationForm(RegistrationForm):
+    gender = forms.ChoiceField(
+        label=_("Gender"),
+        choices=GENDERS,
+        required=True
+    )
+
+    security_question_1_answer = forms.CharField(
```

Comment:
Removed the security questions since Phala is working on a new molo security check.